README.md

```diff
@@ -1,58 +1,144 @@
 ![Banner](Documentation/Images/Banner.png)
 ## Co-op multiplayer RPG and utilities built with Unity Netcode for GameObjects
 
-|[Feedback Form](#feedback-form) |
-| -- |
+Boss Room is a fully functional co-op multiplayer RPG made with Unity Netcode. It is an educational sample designed to showcase typical gameplay [patterns](https://docs-multiplayer.unity3d.com/netcode/current/learn/bossroom/bossroom-actions/index.html) that are frequently featured in similar networked games.
+
+
+__PLEASE NOTE__:
+Boss Room is compatible with the latest Unity Long Term Support (LTS) editor version, currently [2021 LTS](https://unity.com/releases/2021-lts). Please include standalone support for Windows/Mac in your installation. <br>
+
+**Boss Room has been developed and tested on the following platforms:**
```

Comment:
Did we test against console platforms for this release?

Reply:
no